api-reference/introduction.mdx

```diff
@@ -49,3 +49,15 @@ Hyperswitch handles the complex functionality of a comprehensive payments flow t
 | failed                   | The payments object transitions to a failed state when the payment processor confirms the processing failure.                                                                                               |
 | expired                  | You can expire the payments object while it is in any state except when it is under ‘processing’ or ‘succeeded’ state.                                                                                      |
   <img src="assets/images/image.png" />
+
+### Redirect URL
+This is what a url looks like after redirection
+  https://app.hyperswitch.io?status=succeeded&payment_intent_client_secret=pay_NCv9vc19f8aa75OpFxH8_secret_V4zAc7V0C8WAw6FECMKM&amount=10000&manual_retry_allowed=false&signature=4fae0cfa775e4551db9356563d4b98b55662fe3c1c945fe215d90ccf3541282c535909ae901d82174d6b1e46ba1684aa0aa4c8861be0e2a9ef6f950a975d5014&signature_algorithm=HMAC-SHA512
```

Comment:
Should we consider including this within a code block (within triple backticks)?

Also, have we checked how this is rendered by Mintlify: whether it would be wrapped around, or if it would be horizontally scrollable?